pkg/relayer/miner/gen/gen_fixtures.go

```diff
@@ -0,0 +1,334 @@
+// NB: ensure this code is never included in any normal builds.
```

Comment:
I'm not sure I'm following your suggestion. This comment is a build tag which is intended to prevent this file from being included in any build unless `-tags ignore` is used, what I was reffering to as "normal builds". As this is a tag, it must remain a comment, and I don't think we'll ever want to remove it.

Reply:
Can you think of a way to add more clarity to the comment above the build constraint?

Reply:
Ignore me